crates/hotshot/src/tasks/mod.rs

```diff
@@ -168,4 +170,7 @@ pub async fn add_consensus_tasks<
         handle.add_task(QuorumProposalRecvTaskState::<TYPES, I>::create_from(handle).await);
         handle.add_task(Consensus2TaskState::<TYPES, I>::create_from(handle).await);
     }
+
+    #[cfg(feature = "rewind")]
```

Comment:
This is in the `add_consensus_tasks` function. Since it's most relevant to the consensus tasks, it seems like a fair place to put it. Open to put it anywhere else.

Reply:
Other option is to just call `add_rewind_task` where we add all the tests, but this is fine.